src/store/HawkState.ts

```diff
@@ -534,6 +515,36 @@ export function useHawkState(initialSearch?: Partial<Request>): [SearchStore, Se
 		});
 	}
 
+	function getClientData() {
+		let visitId = getCookie('hawk_visit_id');
+		let visitorId = getCookie('hawk_visitor_id');
+
+		if (!visitId) {
+			setCookie('hawk_visit_id', createGuid(), getVisitExpiry());
+			visitId = getCookie('hawk_visit_id');
+		}
+
+		if (!visitorId) {
+			setCookie('hawk_visitor_id', createGuid(), getVisitorExpiry());
+			visitorId = getCookie('hawk_visitor_id');
+		}
+
+        let clientData = {
+			VisitorId: visitorId || '',
+			VisitId: visitId || '',
+			UserAgent: navigator.userAgent,
+			PreviewBuckets: store.searchResults ? store.searchResults.VisitorTargets.map(v => v.Id) : []
+        };
+
+        if (config['language']) {
+            clientData["Custom"] = {
+                "language": config['language']
```

Comment:
Are you sure the language should come under the custom properties? We had a ticket related to this. 
https://hawksearch.atlassian.net/browse/HSC-364

Reply:
Absolutely. This is the data format confirmed by Joanna for enabling the multilingual settings.